docs/release-notes/highlights-2.0.0.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[release-highlights-2.0.0]]
+== 2.0.0 release highlights
+
+[float]
+[id="{p}-200-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.0.0 of {n}. See <<release-notes-2.0.0>> for the full list of changes.
+
+[float]
+[id="{p}-200-CRD"]
+==== Removal of legacy v1beta1 Custom Resource Definitions
+
+ECK `2.0.0` removes support for legacy versions of Kubernetes and no longer ships with a v1beta1 versions of its custom resource definitions. ECK can therefore no longer be installed in versions of Kubernetes <1.16. Please not however that the lowest supported version according to the ECK support policy is `1.19`.
+
+[float]
+[id="{p}-200-topology-zone-awareness"]
+==== Improved support for topology aware Elasticsearch clusters
+Starting with ECK 2.0 the operator can make Kubernetes Node labels available as Pod annotations. This can be used to make information, such as logical failure domains, available in a running Pod. Combined with Elasticsearch shard allocation awareness and Kubernetes topology spread constraints, this makes it much easier then before to create availability zone-aware Elasticsearch clusters.
```

Comment:
```suggestion
Starting with ECK 2.0 the operator can make Kubernetes Node labels available as Pod annotations. This can be used to make information, such as logical failure domains, available in a running Pod. Combined with Elasticsearch shard allocation awareness and Kubernetes topology spread constraints, this makes it much easier than before to create availability zone-aware Elasticsearch clusters.
```